Makefile

```diff
@@ -644,16 +642,14 @@ release-compress: | $(DIST_DIRS)
 	cd $(DIST)/release/; for file in `find . -type f -name "*"`; do echo "compressing $${file}" && gxz -k -9 $${file}; done;
 
 .PHONY: release-sources
-release-sources: | $(DIST_DIRS) npm-cache
+release-sources: | $(DIST_DIRS)
 	echo $(VERSION) > $(STORED_VERSION_FILE)
-	$(eval EXCL := --exclude=$(shell [ ! "$(TAR)" = "tar" ] && echo "^" )./)
-	$(eval EXCL_RECURSIVE := --exclude=)
-	$(TAR) $(EXCL)$(DIST) $(EXCL).git $(EXCL)$(MAKE_EVIDENCE_DIR) $(EXCL_RECURSIVE)node_modules $(EXCL)$(AIR_TMP_DIR) -czf $(DIST)/release/gitea-src-$(VERSION).tar.gz .
+	tar $(addprefix --exclude=./,$(TAR_EXCLUDES)) -czf $(DIST)/release/gitea-src-$(VERSION).tar.gz .
```

Comment:
```suggestion
  $(eval EXCL := --exclude=$(shell tar --help | grep -q bsdtar && echo "^" )./)
	tar $(addprefix $(EXCL),$(TAR_EXCLUDES)) -czf $(DIST)/release/gitea-src-$(VERSION).tar.gz .
```

Reply:
Ugh, GitHub converts between tabs and spaces automatically in commit diff view, but doesn't seem to touch the lines added in suggestion...

Reply:
Is this related to a difference in the exclude syntax? The current syntax produces similar sized tarballs for both `tar (GNU tar) 1.34` and `bsdtar 3.3.2`, so I'm not sure it's necessary.

Reply:
That's because `node_modules` isn't included anymore, so all those `dist` subfolders living there are also spared in both cases. If any folder gets named `dist` in the future they will be excluded by `bsdtar` but included by GNU tar. This discovery was the origin of all hassles around `bsdtar`...

Reply:
Thanks, I'll add a comment why its necessary.